document_page/__manifest__.py

```diff
@@ -43,7 +32,5 @@
     'demo': [
         'demo/document_page.xml'
     ],
-    'installable': False,
-    'auto_install': False,
     'css': ['static/src/css/document_page.css'],
```

Comment:
Yes, please replace this by an XML extending the web client assets to add the intended CSS file.